tests/test-cases/Group23-VIC-Machine-Service/23-04-VCH-Inspect.robot

```diff
@@ -0,0 +1,145 @@
+
+    ${id}=  Get VCH ID  %{VCH-NAME}
+    ${dc-id}=  Get Datacenter ID
+
+    Set Suite Variable  ${VCH-ID}  ${id}
+    Set Suite Variable  ${DC-ID}  ${dc-id}
+
+Teardown
+    Cleanup VIC Appliance On Test Server
+    Terminate All Processes    kill=True
+
+Inspect VCH
+    Get Path Under Target  vch/${VCH-ID}
+
+Inspect VCH Using Session
+    Get Path Under Target Using Session  vch/${VCH-ID}
+
+Inspect VCH Within Datacenter
+    Get Path Under Target    datacenter/${DC-ID}/vch/${VCH-ID}
+
+Inspect VCH Within Datacenter Using Session
+    Get Path Under Target Using Session    datacenter/${DC-ID}/vch/${VCH-ID}
+
+Verify VCH Inspect Output
+    # debug
+    Property Should Be Equal        .debug  1
+
+    # networks
+    Property Should Be Equal        .network.bridge.ip_range                    172.16.0.0/12
+    Property Should Not Be Empty    .network.bridge.port_group
+    Property Should Be Equal        .network.public.nameservers[0]              null
+    Property Should Not Be Empty    .network.public.port_group
+    Property Should Be Equal        .network.container[0].alias                 public
+    Property Should Not Be Empty    .network.container[0].port_group
+
+    # cert
+    ${domain}=  Get Environment Variable  DOMAIN  ''
+    Run Keyword If  $domain != ''   Property Should Contain    .auth.server.certificate.pem  -----BEGIN CERTIFICATE-----
+    Property Should Be Equal        .auth.server.private_key.pem                null
+
+    # compute
+    Property Should Not Be Empty    .compute.resource.id
+
+    # storage
+    Property Should Be Equal        .storage.base_image_size.value              8000000
+    Property Should Be Equal        .storage.base_image_size.units              KB
+
+    Property Should Contain         .storage.image_stores[0]                    %{TEST_DATASTORE}
+    Property Should Contain         .storage.volume_stores[0].datastore         %{TEST_DATASTORE}/%{VCH-NAME}-VOL
+    Property Should Be Equal        .storage.volume_stores[0].label             default
+
+    # ops creds
+    Property Should Be Equal        .endpoint.operations_credentials.user       %{TEST_USERNAME}
+    Property Should Be Equal        .endpoint.operations_credentials.password   null
+
+    # registry
+    Property Should Be Equal        .registry.insecure[0]                       harbor.ci.drone.local
+
+    # connection
+    Property Should Be Equal        .runtime.docker_host                        %{DOCKER_HOST}
+    Property Should Be Equal        .runtime.admin_portal                       %{VIC-ADMIN}
+    Property Should Be Equal        .runtime.power_state                        poweredOn
+    Property Should Contain         .runtime.upgrade_status                     Up to date
+
+    # version
+    ${version}=  Get Version String
+    Property Should Be Equal  .version  ${version}
```

Comment:
nit: as above, I think this would be more readable if `.version` were vertically aligned with the other properties and `${version}` were vertically aligned with the other values